openslides_backend/models/meeting.py

```diff
@@ -16,10 +18,254 @@ class Meeting(Model):
+    agenda_show_internal_items_on_projector = fields.BooleanField()
+
+    list_of_speakers_amount_last_on_projector = fields.IntegerField(minimum=0)
+    list_of_speakers_amount_next_on_projector = fields.IntegerField(minimum=-1)
+    list_of_speakers_couple_countdown = fields.BooleanField()
+    list_of_speakers_show_amount_of_speakers_on_slide = fields.BooleanField()
+    list_of_speakers_present_users_only = fields.BooleanField()
+    list_of_speakers_show_first_contribution = fields.BooleanField()
+
+    motions_default_workflow_id = fields.RequiredForeignKeyField(
+        to=Collection("workflow"), related_name="default_workflow_meeting_id",
+    )
+    motions_default_statute_amendments_workflow_id = fields.RequiredForeignKeyField(
+        to=Collection("workflow"), related_name="default_statute_amendments_meeting_id",
+    )
+    motions_preamble = fields.TextField()
+    motions_default_line_numbering = fields.TextField(
+        enum=["outside", "inline", "none"]
+    )
+    motions_line_length = fields.IntegerField(minimum=40)
+    motions_reason_required = fields.BooleanField()
+    motions_enable_text_on_projector = fields.BooleanField()
+    motions_enable_reason_on_projector = fields.BooleanField()
+    motions_enable_sidebox_on_projector = fields.BooleanField()
+    motions_enable_recommendation_on_projector = fields.BooleanField()
+    motions_show_referring_motions = fields.BooleanField()
+    motions_show_sequential_number = fields.BooleanField()
+    motions_recommendations_by = fields.TextField()
+    motions_statute_recommendations_by = fields.TextField()
+    motions_recommendation_text_mode = fields.TextField(
+        enum=["original", "changed", "diff", "agreed"]
+    )
+    motions_default_sorting = fields.TextField(enum=["weight", "identifier"])
+    motions_identifier_type = fields.TextField(
+        enum=["per_category", "serially_numbered", "manually"]
+    )
+    motions_identifier_min_digits = fields.PositiveIntegerField()
+    motions_identifier_with_blank = fields.BooleanField()
+    motions_statutes_enabled = fields.BooleanField()
+    motions_amendments_enabled = fields.BooleanField()
+    motions_amendments_in_main_list = fields.BooleanField()
+    motions_amendments_of_amendments = fields.BooleanField()
+    motions_amendments_prefix = fields.TextField()
+    motions_amendments_text_mode = fields.TextField(
+        enum=["freestyle", "fulltext", "paragraph"]
+    )
+    motions_amendments_multiple_paragraphs = fields.BooleanField()
+    motions_supporters_min_amount = fields.IntegerField(minimum=0)
+    motions_supporters_enable_autoremove = fields.BooleanField()
+
+    motion_poll_default_type = fields.TextField()  # TODO: PollType enum
+    motion_poll_default_100_percent_base = fields.TextField()  # TODO: PercentBase enum
+    motion_poll_default_majority_method = (
+        fields.TextField()
+    )  # TODO: MajorityMethod enum
+    motion_poll_default_group_ids = fields.ManyToOneArrayField(
```

Comment:
Please remove this setting here and in the list above. We should define this as boolean field in group model.